### PR TITLE
add SCTLR_EL3 register

### PR DESCRIPTION
**Pull request by rmsyn**

### CODE REVIEW

File: src/registers/sctlr_el3.rs [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+        ///
+        /// Possible values of this bit are:
+        ///
+        /// 0b0    Pointer authentication (using the APIAKey_EL1 key) of instruction addresses is not
+        ///        enabled.
+        ///
+        /// 0b1    Pointer authentication (using the APIAKey_EL1 key) of instruction addresses is
+        ///        enabled.
+        ///
+        /// For more information, see Pointer authentication on page D8-5164.
```
Comment (nchong-at-aws) [inline src/registers/sctlr_el3.rs]: What version of the Arm reference are you using? In ARM DDI 0487H.a this page is D5-4775

File: src/registers/sctlr_el3.rs [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+        ///        • Behavior of accessing the banked copies of the stack pointer using the SP register name for
+        ///        loads, stores and data processing instructions.
+        ///
+        ///        • Debug state exit.
+        ///
+        /// The reset behavior of this field is:
+        ///        • On a Warm reset, in a system where the PE resets into EL3, this field resets to an
+        ///        architecturally UNKNOWN value.
+        EIS OFFSET(22) NUMBITS(1) [
+            NoContext = 0,
```
Comment (nchong-at-aws) [inline src/registers/sctlr_el3.rs]: To match EOS field

```suggestion
    NotContextSync = 0,
    ContextSync = 1
```

**Review by nchong-at-aws — APPROVED** [review]
Looks good to me. I can't see why CI failed as the logs have been deleted. If you resubmit then we can respin and get this in.

File: src/registers/sctlr_el3.rs (hunk 1 above)
Comment (rmsyn) [reply src/registers/sctlr_el3.rs]: Not sure, it might have been an older version.

**Pull request merged**